versioned_docs/version-4.x/reference/document-creator/config-file.md

```
@@ -6,9 +6,13 @@ sidebar_label: Configuration File
 
 The configuration file is a JSON file that contains information to configure the application to suit your business needs.
 
+> **⚠️ DISCLAIMER**
+>
+> Configuration files have been deprecated and are not recommended for production use. Configuration files containing private keys should only be used for demo or testing purposes only.
+
 ### Config file structure
 
-A reference of a configuration file can be found [here](https://github.com/TradeTrust/tradetrust-config/blob/master/build/sepolia/config-v2.json).
+AA reference of a configuration file can be found [here](https://github.com/TradeTrust/tradetrust-config/blob/master/build/sepolia/config-v2.json).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)